### PR TITLE
Feature schema

### PR DESCRIPTION
**Pull request by abhiyandhakal**

### CODE REVIEW

File: src/db/schema/goal.ts
Comment (NotSugat) [inline src/db/schema/goal.ts:1]: do we add goal question ?

File: src/db/schema/journal.ts
Comment (NotSugat) [inline src/db/schema/journal.ts:1]: Content from editor js stored as: 
image 
each block is of 
image
Schema needs to be changed accordingly

File: src/db/schema/task.ts
Comment (NotSugat) [inline src/db/schema/task.ts:1]: I think default order needs to be added to swap position later on

**Review by NotSugat — CHANGES_REQUESTED** [review]

File: src/db/schema/journal.ts
Comment (abhiyandhakal) [reply src/db/schema/journal.ts:1]: JSON.stringify op

File: src/db/schema/task.ts
Comment (abhiyandhakal) [reply src/db/schema/task.ts:1]: serial xa

**Pull request merged**